Jellyfin.Plugin.NextPVR/LiveTvService.cs

```diff
@@ -86,38 +87,28 @@ private async Task InitiateSession(CancellationToken cancellationToken)
         {
             _logger.LogInformation("[NextPVR] Start InitiateSession");
             var baseUrl = Plugin.Instance.Configuration.WebServiceUrl;
+            var httpClient = _httpClientFactory.CreateClient(NamedClient.Default);
+            await using var stream = await httpClient.GetStreamAsync(string.Format("{0}/service?method=session.initiate&ver=1.0&device=jellyfin", baseUrl), cancellationToken).ConfigureAwait(false);
+            var clientKeys = new InstantiateResponse().GetClientKeys(stream, _jsonSerializer, _logger);
 
-            var options = new HttpRequestOptions
-            {
-                CancellationToken = cancellationToken,
-                Url = string.Format("{0}/service?method=session.initiate&ver=1.0&device=jellyfin", baseUrl)
-            };
-            options.AcceptHeader = "application/json";
```

Comment:
This header may be needed, depending on how the NextPVR API works. (May return f.ex. XML otherwise)

Reply:
I tested with the new default header, it returns json as expected